project/build.properties

```diff
@@ -1 +1 @@
-sbt.version=1.3.13
+sbt.version=1.4.0
```

Comment:
We should probably do this (from the [release notes](https://github.com/sbt/sbt/releases/tag/v1.4.0)):

> Note to plugin authors
>If you build and publish a plugin using sbt 1.4.0, your users will also be forced to upgrade to sbt 1.4.0 immediately. >To prevent this you can cross build your plugin against sbt 1.2.8 (while using sbt 1.4.0) as follows:
>
> `pluginCrossBuild / sbtVersion := "1.2.8"`

Reply:
since this project uses a custom release process, do you want me to update this line?
```
-  releaseStepCommandAndRemaining(s"^^$latestSbt_1_x_version publish"),
+  releaseStepCommandAndRemaining(s"^^1.3.13 publish"),
```

Reply:
err... yes I think so 😅